sources/packages/api/src-sql/EducationProgramsOfferings/Create-valid-intensity.sql

```diff
@@ -0,0 +1,9 @@
+-- Add the column valid_intensity for Offering with a dummy default value.
+ALTER TABLE sims.education_programs_offerings ADD 
+    COLUMN IF NOT EXISTS valid_intensity sims.valid_intensity NOT NULL DEFAULT 'fullTime';
```

Comment:
I am not able to understand why the name starts with "valid". If for program is "program_intensity" I would expect something like "offering_intensity" or "intensity". Can you please elaborate if there is a reason to call it a "valid_intensity"?

Reply:
offering_intensity is preferred